### PR TITLE
Fix openstack environments

### PR DESCRIPTION
**Pull request by dincho**
The "name" parameter looks like reserved or it's just confusing python calls, so I renamed it to "env_name".

[This is the error](https://travis-ci.org/aeternity/infrastructure/builds/308329315#L659).

Also added dry runs to ensure this is auto checked during travis builds.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 dist: trusty
 language: python
 script:
-  - ansible-lint ansible/environments.yml
+  - openstack stack create test -e openstack/test/create.yml -t openstack/ae-environment.yml --enable-rollback --wait --dry-run
```
Comment (lucafavatella) [inline .travis.yml]: Missing file openstack/test/create.yml ?

Comment (dincho) [reply .travis.yml]: yep here it is 8e513a0, thanks

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 dist: trusty
 language: python
+isntall:
```
Comment (lucafavatella) [inline .travis.yml]: Typo?

Comment (dincho) [reply .travis.yml]: I remove back this build phase (using the default).

File: .travis.yml [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
 language: python
-isntall:
-  - pip install --user -r requirements.txt
 before_script:
   - ansible --version
   - openstack --version
 script:
   - openstack stack create test -e openstack/test/create.yml -t openstack/ae-environment.yml --enable-rollback --wait --dry-run
-  - ansible-playbook --check -i localhost, ansible/environments.yml
+  - ansible-playbook --check -i localhost, -e 'ansible_python_interpreter="/usr/bin/env python"' ansible/environments.yml
```
Comment (lucafavatella) [inline .travis.yml]: Was it defaulting to Python3? What was the symptom?

Comment (dincho) [reply .travis.yml]: It looks like Ansible has [hardcoded python path](https://github.com/ansible/ansible/issues/6345#issuecomment-37107657) to system python, but travis is using virtualenv and [won't find the installed modules](https://travis-ci.org/aeternity/infrastructure/builds/308395778#L990)

**Pull request merged**